doc/workshops/financial.rst

```diff
@@ -2,4 +2,1065 @@
+
+.. code-block:: python
+
+   pprint.pprint(netflix_info)
+
+Although ``pprint`` displays all the information, it is easier to view it in the Variable Explorer.
+
+
+Historical stock data
+~~~~~~~~~~~~~~~~~~~~~
+
+We can download in a dataset the history of a stock with the following line of code:
+
+.. code-block:: python
+
+   hist = netflix.history(period="max")
+
+We can see a summary and more details of this dataset in the Variable Explorer. It appears as a DataFrame object, with 7 columns and thousands of rows. If we double-click on it we will see the historical records of Netflix stock organized by ascending dates.
+
+.. image:: images/workshop-1/netflix-history.png
+   :alt: Netflix historical operations
+
+Throughout this workshop we will use this historical information to compare, with various normative financial theories, different types of financial portfolios.
+
+
+
+===============
+First portfolio
+===============
+
+Let's build our first stock investment portfolio! Say we are interested in investing in technology, and we want to know what performance can be obtained by putting money into some of the "heavyweights" in this industry.
+
+To measure the performance of our first portfolio we are going to use a classic theory in the world of finance: *mean-variance portfolio (MVP) theory*. This model assumes that investors only care about expected returns and the variance of such returns. The analysis is based entirely on statistical measures based on a time series of share prices, such as periodic mean returns and the variances of those returns with the same periodicity.
+
+
+Prepare portfolio data
+~~~~~~~~~~~~~~~~~~~~~~
+
+Before we start, let's run some lines of code to style the plots.  These lines are optional, but we recommend that you run them so that the graphics look like the screenshots we present in this workshop.
+
+.. code-block:: python
+
+   plt.style.use("fivethirtyeight")
+   mpl.rcParams["savefig.dpi"] = 300
+   mpl.rcParams["font.family"] = "serif"
+   np.set_printoptions(precision=5, suppress=True, formatter={"float": lambda x: f"{x:6.3f}"})
+
+Suppose we want to measure the performance of a portfolio consisting of Google, Apple, Microsoft, Netflix and Amazon stocks. Let's call this set ``SYMBOLS_1``.
+
+.. code-block:: python
+
+   SYMBOLS_1 = ["GOOG", "AAPL", "MSFT", "NFLX", "AMZN"]
+
+.. note::
+
+   If you search for ``SYMBOLS_1`` in Variable Explorer you will not find it: Python interprets this element not as a variable, but as a **constant**. This is because the name is written with uppercase letters (no letter is lowercase).
```

Comment:
By default the Variable Explorer doesn't show this but it would be cool to mention that they can actually change the settings in the options menu to be able to see this variable.